_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,213 @@
+		<article style="font-size: 16px; word-spacing: 1.8pt !important; line-height: 1em !important;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</article>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `p` element has a `line-height` of `30px !important` (equals `1.875em`) which is above the permissible minimum, given the default pixel size of the body is 16 pixels.
+
+```html
+<p style="line-height: 30px !important">
+	The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+	sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+</p>
+```
+
+#### Failed Example 2
+
+This `div` element has a `letter-spacing` of `1em !important` which is above to the permissible minimum, given the default pixel size of the body is 16 pixels.
+
+```html
+<div style="letter-spacing: 1em !important;">
+	The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+	sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+</div>
+```
+
+#### Failed Example 3
+
+This `strong` element has a `word-spacing` of `8px !important` (equals `0.5em`) which is above the permissible value, given the default pixel size of the body is 16 pixels.
+
+```html
+<strong style="word-spacing: 8px !important;">
+	The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+	sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+</strong>
+```
+
+#### Failed Example 4
+
+This `article` element has both `letter-spacing` and `line-height` specified, where `line-height` is above the permissible value, given the default pixel size of the body is 16 pixels.
+
+```html
+<article style="letter-spacing: 0.12em !important; line-height: 3em !important;">
+	The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+	sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+</article>
+```
+
+#### Failed Example 5
+
+This `p` element has a `line-height` of `1.25em !important`, which is greater than `1.5em` given the body has a default pixel size of 24 pixels.
+
+```html
+<html>
+	<body style="font-size: 24px;">
+		<p style="line-height: 1.25em !important">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</p>
+	</body>
+</html>
+```
+
+### Inapplicable
```

Comment:
We should add an inapplicable example where "important" is used on one of those props on a style element.